renderlib/Logging.cpp

```diff
@@ -68,3 +76,38 @@ Logging::Enable(bool enabled)
 {
   boost::log::core::get()->set_logging_enabled(enabled);
 }
+
+boost::filesystem::path

```

Comment:
Targeting C++ 14.  Because I'm pulling in other git submodules with their own cmake files, and also I'm already depending on boost for other things, it was simpler to just use boost for this rather than introduce C++17.